src/lib/components/tablev2/Tablestyle.js

```diff
@@ -26,7 +29,10 @@ export const TableHeader = styled.div`
 
 export const HeadRow = styled.div`
   height: 2.286rem;
-  width: 100% !important;
+  width: ${(props) =>
+    props.hasScrollBar
+      ? `calc(100% - ${props.scrollBarWidth}px - ${borderSize} )!important;` // -4px for border
+      : '100% !important;'} 
```

Comment:
nit: I don't think that the hasScrollBar props is necessary, we can probably do :
```suggestion
  width: ${(props) =>
    props.scrollBarWidth
      ? `calc(100% - ${props.scrollBarWidth}px - ${borderSize} )!important;` // -4px for border
      : '100% !important;'} 
```

Then in src/lib/components/tablev2/SingleSelectableContent.js the state could be removed

Reply:
we can not do that :) 

as we discussed, the scrollbarWidth is calculated by creating a new div and getting the scroll bar size : https://github.com/scality/core-ui/pull/400/files#diff-c49bb2bd3fc72cb8413901b8bf5b5f6ce38a75db48670f24165ee4e41bc18e26R152-R163

so we always have scrollbarWidth even if the scrollBar is not displayed. So we need to know if the scrollBar is displayed or not with this props.

Reply:
Oh yes that's right 👍